### PR TITLE
Field addition

### PR DESCRIPTION
**Pull request by tjasmith**
Removed methods to get the values of the extra fields
Updated the methods to handle the type of the extra fields data (JSONArray)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -370,27 +370,12 @@ This license is based on the Apache Software License, version 1.1.</extractedTex
...
-					<execution>
-						<id>attach-javadocs</id>
-						<configuration>
-							<additionalparam>${javadoc.opts}</additionalparam>
-						</configuration>
-						<goals>
-							<goal>jar</goal>
-						</goals>
-					</execution>
-				</executions>
```
Comment (goneall) [inline pom.xml:392]: @tjasmith Why did you remove the extra execution in the POM file?

File: src/org/spdx/rdfparser/license/SimpleLicensingInfo.java [hunk 2]
```diff
@@ -44,18 +46,20 @@
 	protected String comment;
 	protected String name;
 	protected String[] seeAlso;
-	
+	// protected String[] seeAlsoDetails;
+	protected JSONArray seeAlsoDetails;
```
Comment (goneall) [inline src/org/spdx/rdfparser/license/SimpleLicensingInfo.java:50]: Minor comment - looks like the comment above could be removed

File: src/org/spdx/rdfparser/model/RdfModelObject.java [hunk 3]
```diff
@@ -436,6 +438,28 @@ protected void setPropertyValue(String nameSpace, String propertyName,
 		}
 	}
 	
+	protected void setPropertyValue(String nameSpace, String propertyName,
```
Comment (goneall) [inline src/org/spdx/rdfparser/model/RdfModelObject.java:441]: This is OK - but I would prefer to translate the JSONArray to a Java array of values outside of the RdfModelObject to reduce the dependency graph for the RdfModelObject.  If the current caller(s) of this method translated into a standard Java object or primitive type, it would be a cleaner design IMHO.

File: src/org/spdx/rdfparser/license/SimpleLicensingInfo.java (hunk 2 above)
Comment (tjasmith) [reply src/org/spdx/rdfparser/license/SimpleLicensingInfo.java:50]: Thanks for catching that

File: pom.xml (hunk 1 above)
Comment (tjasmith) [reply pom.xml:392]: I removed it because with my java version, the javadoc executable is not in JAVA_HOME/jre/bin/ . Which jdk version do you use?
